src/Styleguide/Elements/RadioGroup.tsx

```diff
@@ -63,22 +53,11 @@ export class RadioGroup extends React.Component<
   }
 
   render() {
+    const { disabled, onSelect, defaultValue, children, ...others } = this.props
```

Comment:
Just to verify I understand this, we have assigned `others` to the values in `this.props` except for `disabled`, `onSelect`, etc? This is a really cool pattern! Does it have a name?

Reply:
😄